### ISSUE TITLE
Normalization of input images

### ISSUE THREAD

**Issue by zzg-zzg** [aipixel/GPS-Gaussian#44]
Dear authors,
I wonder if the input image data has been standardized preprocessed (e.g. mean_vals = [0.471, 0.448, 0.408] std_vals = [0.234, 0.239, 0.242]). I can't find the relevant code. 
If the answer is no, I wonder if it would be beneficial to do so, since many pretrained models use standardization.
Thanks

**Comment by ShunyuanZheng** [comment]
I simply normalize the RGB value to [-1, 1] since I do not load the pre-trained weights trained on typical datasets for 2D vision tasks (_e.g._ Imagenet, COCO, etc). I think there is no significant difference between the pattern of standardization when training a model from scratch.

**Comment by zzg-zzg** [comment]
Thanks for your reply.
